src/posix/platform/radio.cpp

```diff
@@ -205,6 +209,56 @@ void Radio::Init(void)
 
 void *Radio::GetSpinelInstance(void) { return &sRadioSpinel; }
 
+void Radio::GetIidListFromRadioUrl(spinel_iid_t *aIidList, const uint8_t aLength)
```

Comment:
Can pass a reference to array here (I dont think we will ever have any use-case where we may want to use a different array size here for this method)?

```c++
void Radio::GetIIdListFromRadioUrl(spinel_iid_t (&aIIdList)[Spinel::kSpinelHeaderMaxNumIID]));
```

Reply:
Sounds good 👍 .